Cargo.toml

```diff
@@ -16,12 +16,15 @@ base64 = "0.12.3"
 rand = "0.7"
 ring = "0.16.15"
 thiserror = "1.0"
+serde = { version = "1.0", features = ["derive"] }
+
```

Comment:
nit: Remove this `\n`, since the other headers in this file have a single new line after them.